dl/stubs.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build darwin
+// +build darwin
+
+package dl
+
+var dlopenABI0 uintptr
```

Comment:
The rest of these are here for future use. Do we want to remove them now or keep them for later?

Reply:
Let's remove unused ones. I think dlopen and dlsym are used. Is that correct?

Reply:
Currently, only dlsym is used but dlopen will be needed very soon to open glfw.dylib and other frameworks. Should I add dlopen to the exported functions?

Reply:
Yeah, if we need dlopen very soon, it's ok to add this now.